react/components/CreditCard.tsx

```diff
@@ -267,11 +280,16 @@ const CreditCard: React.FC = () => {
         />
       </div>
       <div className="flex mt5">
-        <Button block disabled={saveLoading} onClick={handleSubmit}>
+        <Button
+          size="large"
+          block
+          disabled={saveLoading}
+          onClick={handleSubmit}
+        >
           {saveLoading ? (
             <Spinner size={24} />
           ) : (
-            <FormattedMessage id="checkout-payment.button.save" />
+            intl.formatMessage(messages.saveInfoLabel)
```

Comment:
even though the button is large, our font-size should be set to 1rem

```suggestion
            <span className="f5">{intl.formatMessage(messages.saveInfoLabel)}</span>
```